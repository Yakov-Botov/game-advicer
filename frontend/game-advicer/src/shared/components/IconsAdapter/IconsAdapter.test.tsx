import React from 'react';
import { render } from '@testing-library/react';
import {IconsAdapter} from "."

test('Рендеринг IconsAdapter компонента', () => {
    const {container} = render(<IconsAdapter iconType={"Icon28LikeOutline"} />);
    expect(container).toMatchSnapshot()
})