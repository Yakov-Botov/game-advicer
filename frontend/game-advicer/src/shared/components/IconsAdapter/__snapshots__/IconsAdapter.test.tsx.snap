// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Рендеринг IconsAdapter компонента 1`] = `
<div>
  <div
    class="Icon Icon--28 Icon--w-28 Icon--h-28 Icon--like_outline_28 "
    style="width: 28px; height: 28px;"
  >
    <svg
      height="28"
      style="display: block;"
      viewBox="0 0 28 28"
      width="28"
    >
      <use
        style="fill: currentColor;"
        xlink:href="#like_outline_28"
      />
    </svg>
  </div>
</div>
`;
